fabtests/pytest/efa/test_rma_pingpong.py

```diff
@@ -19,6 +19,11 @@ def test_rma_pingpong(cmdline_args, iteration_type, operation_type, completion_s
 
 @pytest.mark.functional
 @pytest.mark.parametrize("operation_type", ["writedata", "write"])
+@pytest.mark.parametrize("message_size", ["r:1,4,65",
+                                          "r:4048,4,4148",
+                                          "r:8000,4,9000",
+                                          "r:17000,4,18000",
+                                          "r:1,1024,1048577"])
```

Comment:
I'm not sure if we want `r:1,4,65` - it expands to `1,5,9,13,...` which does not look right.

Overall I think it's probably easier to skip the test if message size starts from 0.

Reply:
so we just test "r:4048,4,4148",
                          "r:8000,4,9000",
                          "r:17000,4,18000"?